packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/OutputTypes/ScmOutputLibrary.cs

```diff
@@ -11,7 +11,16 @@ public class ScmOutputLibrary : OutputLibrary
     {
         protected override IReadOnlyList<TypeProvider> BuildTypes()
         {
-            return [.. base.BuildTypes(), new ModelSerializationExtensionsProvider(), new TypeFormattersProvider()];
+            var baseTypes = base.BuildTypes();
```

Comment:
I think we should update BuildTypes to return a TypeProvider[], then you can simply replace the item you want to replace.  The public Types property should still return an IReadOnlyList